README.md

```diff
@@ -34,7 +35,8 @@ Welcome to the *Chat with your data* Solution accelerator repository! The *Chat
 
 ### About this repo
 
-This repository provides a template for setting up the solution accelerator, along with detailed instructions on how to use and customize it to fit your specific needs. It provides the following features:
+This repository provides a template for setting up the solution accelerator, along with detailed instructions on how to use and customize it to fit your specific needs. This repository is created to provide an end-to-end solution accelerator for users who want to use natural language to query their data. The repo includes a well designed ingestion mechanism, best practises for deployment, and a support team for maintenance. This repository also helps in creating solution for ingesting multiple data types, and it uses the dataSources parameter which allows diffrent data types for ingestion, with a vision where with single front end you were able to use multiple backends with Azure, Openai or the custom back end. 
+The simpler frontend of the chat-with repo seems to have streamlined the interaction with Multiple backend a lot.  It provides the following features:
```

Comment:
I don't quite understand what this sentence is trying to say?

Reply:
statement is modified can you please review the changes again

Reply:
I think the sentence that @cecheta is referring to is this: - `The simpler frontend of the chat-with repo seems to have streamlined the interaction with Multiple backend a lot.` - I think you should just delete it